varcode/variant_annotator.py

```diff
@@ -11,7 +12,7 @@ class VariantAnnotator(object):
     def __init__(self, ensembl_release):
         self.ensembl = pyensembl.EnsemblRelease(ensembl_release)
 
-    def describe_variant(self, variant):
+    def describe_variant(self, variant, raise_on_error=True):
```

Comment:
Yeah, I figured `VariantCollection` would pass it through to `VariantAnnotator`. The constructor just felt more appropriate to me, based on knowing far less about the code than you do.